tribits/ci_support/cdash_analyze_and_report.py

```diff
@@ -288,6 +302,11 @@ def setExtraCmndLineOptionsAfterParse(inOptions_inout):
   else:
     setattr(inOptions_inout, 'printDetails', False)
 
+  if inOptions_inout.listUnexpectedBuildsStr == "on":
+    setattr(inOptions_inout, 'listUnexpectedBuilds', True)
+  else:
+    setattr(inOptions_inout, 'listUnexpectedBuilds', False)
```

Comment:
If this were me, I'd write it as
```python
setattr(inOptions_inout, 'listUnexpectedBuilds', inOptions_inout.listUnexpectedBuildsStr == "on")
```

Reply:
That is a good suggestion.  I will try that out.

Reply:
That seems to work like a charm.  I have made this change in my local branch.